scripts/compile-dependencies

```diff
@@ -26,6 +26,7 @@ for PKG in squashfs-tools squashfuse e2fsprogs fuse-overlayfs gocryptfs; do
     fi
     DIR=${TGZ%.tar.gz}
     DIR=${DIR/*\//}
+    VER="${DIR/*-/}"
```

Comment:
This isn't actually going to work for the packages with a hyphen in the name.  I suggest just moving it down to the gocryptfs case.